tests/test_hyperbolic.py

```diff
@@ -334,32 +335,21 @@ def test_dist_poincare(self):
         point_a = gs.array([0.5, 0.5])
         point_b = gs.array([0.5, -0.5])
 
-        self.space.metric.coords_type = 'ball'
-
-        dist_a_b = self.metric.dist(point_a, point_b)
-        self.space.metric.coords_type = 'extrinsic'
+        dist_a_b =\
+            self.ball_manifold.metric.dist(point_a, point_b)
 
         result = dist_a_b
         expected = gs.array([[2.887270927429199]])
 
         self.assertAllClose(result, expected)
 
-    def test_exp_poincare(self):
```

Comment:
Where did this test go?

Reply:
Maybe it's in the file `test_poincare_ball.py` -> in this case, put test_log_poincare there, too?